source/documentation/using_the_api/register_components.md

```diff
@@ -12,21 +12,23 @@ Entries contain metadata about what and when data has changed.
 
 Entries contain the fields `entry-number`,  `entry-timestamp`, `item-hash` and `key`. 
 
-The `entry-number` is unique and defines an entry’s position within the ordered list of a register. The `entry-timestamp` is the time when an entry was introduced to a given register, but it is no guarantee of the order of entries in a register. The `entry-number` is.
+The entry number is unique and defines an entry’s position within the ordered list of a register. The entry timestamp is the time when an entry was introduced to a given register, but it is no guarantee of the order of entries in a register. The entry number is.
```

Comment:
Re-reading this paragraph, perhaps it's worth clarifying that the "ordered list" is actually the "ordered list of changes"? So no one confuses the list of entries with the list of records.